utils.py

```diff
@@ -171,11 +171,10 @@ def _generate_metadata(dataset):
                 if masks_dir_name.startswith(masks_dir_prefix):
                     masks_overlayed_sufix_to_write.append(masks_dir_name[len(masks_dir_prefix):])
 
-        df_columns = ['ImageId', 'file_path_image', 'is_train', 'is_valid', 'is_test', 'n_buildings']
+        df_dict = {'ImageId': [], 'file_path_image': [], 'is_train': [], 'is_valid': [], 'is_test': [], 'n_buildings': []}
```

Comment:
@taraspiotr it is recommended to do something with either setdefault method or
use defaultdict object 

```
df_dict={}
for ...
   df_dict.setdefault('bla',[]).append(cool_value)
```